tool/site/server_hapi/src/services/communication/live-eps-client.ts

```diff
@@ -32,6 +32,18 @@ export class LiveEpsClient implements EpsClient {
     return {statusCode, fhirResponse, spineResponse}
   }
 
+  async makeReleaseRequest(body: Parameters): Promise<EpsResponse<Bundle | OperationOutcome>> {
+    const requestId = uuid.v4()
+    const rawResponseHeaders = {
+      "x-raw-response": "true"
+    }
+    const response = await this.makeApiCall<OperationOutcome>("Task/$release", body, requestId)
```

Comment:
`Bundle | OperationOutcome`? Not that it matters evidently!

Reply:
good spot

Reply:
updated on next pr